2.5

```diff
@@ -1,38 +1,42 @@
-# 2.5 Exercise
-# Jimmy Hattier, Diontre Davis, and Yvonne Ma
-# 1/23/24
+# Exercise 2
+# Yvonne Ma
 
-# 1) Why does this code not work?
+# 1. Why does this code not work?
 
 my_variable <- 10
 my_varıable
 
-# The object "my_varıable" has the wrong character for "i", instead using "ı"
+# The object "my_varıable" is not found, because it uses "ı" instead of "i", so in R's view there only exists "my_variable", not "my_varıable",
+# and R cannot recognize it
 
-# 2) Tweak each of the following R commands so that they run correctly:
+# 2. Tweak each of the following R commands so that they run correctly:
 
 libary(todyverse)
 
 ggplot(dTA = mpg) + 
   geom_point(maping = aes(x = displ y = hwy)) +
   geom_smooth(method = "lm)
 "
-# ^I have to close the open quotation marks before I can code the correction!
+
+# Correct as follows:
+
 library(tidyverse)
 
-ggplot(data = mpg) +
-  geom_point(mapping = aes(x = displ, y = hwy)) +
-  geom_smooth(aes(x = displ, y = hwy), method = "lm")
-# The code is littered with errors, including misspelling,
-# improper capitalization, and missing punctuation.
+ggplot(
+  data = mpg,
+  mapping = aes(x = displ, y = hwy)) +
+  geom_point()+
+  geom_smooth(method = "lm")
+
+# Do we have to do "  geom_smooth(aes(x = displ, y = hwy), method = "lm")"?
+# can we omit aes(x,y) inside geo_smooth as above?
```

Comment:
Because mapping wasn't specified at the global layer, it needs to be specified for all sublayers.